groups/script.js

```diff
@@ -264,10 +283,26 @@ async function addrole() {
 /**
  * TO REMOVE YOURSELF OF A ROLE
  */
+
 async function removeRoleHandler() {
-  console.log('Remove function to be added after this pr');
+  if (memberAddRoleBody?.userid && memberAddRoleBody?.roleid !== '') {
+    loader.classList.remove('hidden');
 
-  // TODO: REMOVE ME BUTTON FUNCTIONALITY TO BE ADDED
+    try {
+      // Remove the role from the member
+      const res = await removeRoleFromMember(
+        memberAddRoleBody.roleid,
+        memberAddRoleBody.userid,
+      );
+      showToaster(res.message);
+      UserGroupData = await getUserGroupRoles();
+      updateButtonState();
+    } catch (err) {
+      console.error(err);
```

Comment:
Remove console please

Reply:
Actually this format of console.error has been used in many places in dashboard site by other developers so I tend to follow it.